src/sidebar/components/test/annotation-body-test.js

```diff
@@ -88,6 +89,27 @@ describe('AnnotationBody', () => {
     assert.equal(buttonProps.title, 'Show the first few lines only');
   });
 
+  describe('tag list and editor', () => {
+    it('renders a list of tags if not editing and annotation has tags', () => {
+      const wrapper = createBody({ isEditing: false, tags: ['foo', 'bar'] });
+
+      assert.isTrue(wrapper.find('TagList').exists());
+    });
+
+    it('does not render a tag list if annotation has no tags', () => {
+      const wrapper = createBody({ isEditing: false, tags: [] });
+
+      assert.isFalse(wrapper.find('TagList').exists());
+    });
+
+    it('renders a tag editor if annotation is being edited', () => {
+      const wrapper = createBody({ isEditing: true, tags: ['foo', 'bar'] });
+
+      assert.isTrue(wrapper.find('TagEditor').exists());
+      assert.isFalse(wrapper.find('TagList').exists());
+    });
+  });
+
   it(
     'should pass a11y checks',
```

Comment:
I know it was missing before, but can we add one with a tag list length > 0

And It could also be good to get a state where `.annotation-body__collapse-toggle` is rendered. e.g 
```isCollapsible && !isEditing```

Reply:
> I know it was missing before, but can we add one with a tag list length > 0

Do you mean an accessibility test?

Reply:
test